sdk/rpc/server/server.go

```diff
@@ -148,13 +149,9 @@ func (t *RPCServer) CredentialProvisionerProvision(req proto.ProvisionCredential
 	if err != nil {
 		return err
 	}
-	*resp = sdk.ProvisionOutput{
-		Environment: make(map[string]string),
-		CommandLine: nil,
-		Files:       make(map[string]sdk.OutputFile),
-		Diagnostics: sdk.Diagnostics{},
-	}
+	*resp = req.ProvisionOutput
 	provisioner.Provision(context.Background(), req.ProvisionInput, resp)
+
```

Comment:
tinny tiny nit: Let's remove this newline for consistency 😄

Reply:
If this is going to reset all the approvals, feel free to ignore this.

Reply:
```suggestion
```